src/main/kotlin/gdx/liftoff/data/templates/Template.kt

```diff
@@ -344,6 +345,41 @@ public class IOSLauncher extends IOSApplication.Delegate {
     }
 }"""
 
+  fun addIOSMOESVMRegistration(project: Project) {
+    addSourceFile(
+      project = project,
+      platform = IOSMOE.ID,
+      packageName = project.basic.rootPackage,
+      fileName = "SVMRegistrationFeature.$launcherExtension",
+      content = getIOSMOESVMRegistrationContent(project),
+    )
+  }
+
+  fun getIOSMOESVMRegistrationContent(project: Project): String =
+    """package ${project.basic.rootPackage};
+
+import org.graalvm.nativeimage.hosted.Feature;
+import org.graalvm.nativeimage.hosted.RuntimeJNIAccess;
+
+import java.nio.Buffer;
+import java.nio.ByteBuffer;
+import java.nio.CharBuffer;
+import java.nio.DoubleBuffer;
+import java.nio.FloatBuffer;
+import java.nio.IntBuffer;
+import java.nio.LongBuffer;
+import java.nio.ShortBuffer;
+
+public class SVMRegistrationFeature implements Feature {
```

Comment:
I usually like to have any generated code I'm putting into user projects have at least a little documentation that tells users why it's there and whether they might need to edit/delete it. I will merge this probably tonight, since I want to get started on updating to Gradle 9.x, but some advice on how to describe this file would be helpful (maybe a link?).